float64 a_x
float64 a_y
float64 v_z
float64 yaw_rate